resources/baseline-config.json

```diff
@@ -0,0 +1,755 @@
+{
```

Comment:
I've run these files through a tool called json_pp (JSON pretty print) that I found on my machine. I'm not sure this format is the best, but it's very diff-able.